### ISSUE TITLE
5345084D - SONIC FREE RIDERS

### ISSUE THREAD

**Issue by Resilixia** [xenia-project/game-compatibility#1125]
[Marketplace](https://marketplace.xbox.com/en-US/Product/SONIC-FREE-RIDERS/66acd000-77fe-1000-9115-d8025345084d)

Tested on (https://github.com/xenia-project/xenia/commit/dc0b468ea81157bdc50be4159c890d30229f4000)

# Issues:
(Introduction video loops due to no Kinect support to access title screen.)

# Log:
[xenia.log](https://github.com/xenia-project/game-compatibility/files/2698700/xenia.zip)

# Screenshot(s):
![Black screen after Intro clip, audio can be eared asking to move the Kinect cursor](https://i.imgur.com/PjWiNa4.png)

# Labels:
state-intro

**Comment by ZolaKluke** [comment]
Tested on https://github.com/xenia-project/xenia/commit/9418f3f2c5fb3b98421cca7f71506bc02fa48735

### Issues:
Game gets to the title screen however it requires a kinect to get past it and so ends up looping the intro video, also to get the title screen and the logos before the intro video to show correctly `gpu_allow_invalid_fetch_constants` must be set to `true`

### Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/4932021/xenia.zip)

### Screenshot(s):
![image](https://user-images.githubusercontent.com/6497775/87682033-b73c8100-c7c2-11ea-957e-80100b65e13c.png)
![image](https://user-images.githubusercontent.com/6497775/87682047-bb689e80-c7c2-11ea-8220-32988caeb133.png)
![image](https://user-images.githubusercontent.com/6497775/87682169-ddfab780-c7c2-11ea-900e-684357cb1cf5.png)
![image](https://user-images.githubusercontent.com/6497775/87681969-a25fed80-c7c2-11ea-8977-9d2914228e93.png)

### Labels:
state-title

**Comment by Megasonic999** [comment]
If only it had a "switch to kinect to controller" patch or something like that.

**Comment by Sonic3AndKnux** [comment]
yeah so free riders is unplayable until xenia can emulate a kinect using a controller

**Comment by MilesTailsFan2005** [comment]
Their is a concept Patch by one person which is bringing life into the game until xenia finds away to get The Kinect Camera support

https://youtu.be/e65ALtDVin8

**Comment by icup321** [comment]
The No-Kinect Patch mentioned by @MilesTailsFan2005 has been released:
[https://www.youtube.com/watch?v=xSJu_QK3kSY](url)
According to the readme, it requires you to set gpu_allow_invalid_fetch_constants to true and set protect_zero to false. There also emulation bugs where the game can softlock with a black screen for unknown reasons.

**Comment by MilesTailsFan2005** [comment]
> The No-Kinect Patch mentioned by @MilesTailsFan2005 has been released: [https://www.youtube.com/watch?v=xSJu_QK3kSY](url) According to the readme, it requires you to set gpu_allow_invalid_fetch_constants to true and set protect_zero to false. There also emulation bugs where the game can softlock with a black screen for unknown reasons.

Ah OK thanks for letting me know

**Comment by Kirby0Louise** [comment]
As of v1.1 of the No Kinect mod, it's noted that VirtualProtectEx does not work in Xenia [per the mod's source code](https://github.com/ReimousTH/FreeAPI/blob/master/HookV2.cpp#L22), and results in an emulator detection

**Comment by DetectiveAlice** [comment]
i've noticed GUI errors in xenia using this patch, but i can't confirm if it's patch related or xenia related. the errors only appear sometimes.

other visual glitches happen frequently as well, but otherwise the game plays just fine.
